web-common/src/features/sources/navigation/SourceMenuItems.svelte

```diff
@@ -119,7 +116,7 @@
       return;
     }
     try {
-      await refreshSource(connector, tableName, runtimeInstanceId);
+      await refreshSource(connector, tableName, sourceName, runtimeInstanceId);
```

Comment:
Here I see `tableName` is passed in as the `filePath` parameter. Is that a mistake?

Reply:
Yes it should be filePath